src/Serilog.Extensions.Hosting/SerilogHostBuilderExtensions.cs

```diff
@@ -130,8 +187,8 @@ public static class SerilogHostBuilderExtensions
 
                     return factory;
                 });
-
-                ConfigureServices(collection, logger);
+                
+                ConfigureServices(collection, null);
```

Comment:
Why is there now `null`?

Reply:
Thanks for the note, @sungam3r - it's because `ILogger` is already registered lazily above (line 156); I've added a comment :+1: